### ISSUE TITLE
on centos7.6, when run client, all the network lost connection

### ISSUE THREAD

**Issue by morganwang010** [jedisct1/dsvpn#55]
when I run the dsvpn client on the centos7.6, it show create tun0 port,  and then lost all the network connection. Even if I restart the network service , but it can not connect the normal network, so I have to reboot the system.
Is there some condtion need to run the client?

**Comment by jedisct1** [comment]
What exactly is being displayed, both server-side and client-side?

**Comment by morganwang010** [comment]
> What exactly is being displayed, both server-side and client-side?
I run the client-side at the virtualbox vms. And it occurs only on the client-side. The server-side is ok. 
at client-side , the route list, the ip address , looks all good. Sometime, the tun0 iface was not created. When ping a ipaddress which is at the same subnet of the iface, it says network is unreachable. seems the network was broken  .

**Comment by morganwang010** [comment]
moreover, I occasionally make the client connects to server successfully. but when I use the command telnet google.com 80  at the client-side, the servier-side suddenly quit . May be something wrong with my virtualbox vms.

**Comment by jedisct1** [comment]
You shouldn't have to manually add routes.

If you type `route del default`/`route add default dev tun0` (does the `route` command still exist on Linux?), there are no more routes for the machine to communicate with the server.

Without knowing what is being displayed server-side and client-side, it's unfortunately impossible to help.

**Comment by dazuimao1990** [comment]
I had the same problem,When I started dsvpn_client, the connection to my gateway is unreachable.
This led to the dsvpn_client can not connect to the server.the tun0 interface have no ip address.
the only way to repair my network is reboot the OS.
![image](https://user-images.githubusercontent.com/33855684/69291748-20252680-0c3f-11ea-8d11-4940ce0ec476.png)
![image](https://user-images.githubusercontent.com/33855684/69291935-a04b8c00-0c3f-11ea-82a4-b913b52575f2.png)

**Comment by dazuimao1990** [comment]
how about reopen this issue? I do not know what is going wrong with my dsvpn client. waiting for help.

**Comment by jedisct1** [comment]
Unfortunately, I don't know much about Linux and I've never used CentOS so I won't be able to help. But since you do, I'm waiting for your help.

**Comment by dazuimao1990** [comment]
OK，I'm still trying.
First, I tried to upgrade all the packages,with the command `yum update`. It doesn't work.
and then,I had upgrade my kernel,I admit I didn't notice the requirement for a higher kernel version.
for this time,I can ping the gateway.but lost Internet connection.I don't know how to continue.

Can you provide an available environment configuration information. Include Linux distributions, kernel versions, virtual machines, or physical machines.

Thank you, my friend.

**Comment by dazuimao1990** [comment]
hello, I tried something new.
I found that traffic to the Internet now passes through the `tun0` interface, which makes the Internet inaccessible.

![image](https://user-images.githubusercontent.com/33855684/69697221-d9db3600-111c-11ea-9366-6e921a65c505.png)

`tun0` is inited by dsvpn_client.

![image](https://user-images.githubusercontent.com/33855684/69697354-29b9fd00-111d-11ea-958e-734eb4425115.png)

when I stop the dsvpn_client.the Internet connection came back.
